commit 36b7bbbded6cb372b47d5374988c6ffe0cca3f24 (refs/changes/14/1514/1)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2017-02-15T11:32:17-08:00 (2 years, 8 months ago)
    
    AGENT-1058 logadm calls in agents -setup scripts could use -v for more verbosity

diff --git a/smf/method/vm-agent-setup b/smf/method/vm-agent-setup
index 0aa1083..cf005cb 100755
--- a/smf/method/vm-agent-setup
+++ b/smf/method/vm-agent-setup
@@ -21,7 +21,7 @@ set -o xtrace
 
 function setup_logadm {
     mkdir -p -m 0755 /var/log/vm-agent
-    logadm -w vm_agent_logs -C 168 -S 1g -c -p 1h \
+    logadm -v -w vm_agent_logs -C 168 -S 1g -c -p 1h \
         -t '/var/log/vm-agent/vm-agent_$nodename_%FT%H:%M:%S.log' \
         /var/svc/log/*vm-agent*.log
 
@@ -29,8 +29,8 @@ function setup_logadm {
     # that the default '-C' (from
     # https://github.com/joyent/smartos-live/blob/master/overlay/generic/etc/logadm.conf)
     # doesn't defeat our attempts to rotate out of "/var/svc/log".
-    /usr/sbin/logadm -r smf_logs
-    /usr/sbin/logadm -w smf_logs -C 8 -c -s 1m '/var/svc/log/*.log'
+    /usr/sbin/logadm -v -r smf_logs
+    /usr/sbin/logadm -v -w smf_logs -C 8 -c -s 1m '/var/svc/log/*.log'
 }
 
 
